docs/interactive-mode.md

```diff
@@ -0,0 +1,179 @@
+# Interactive Mode
+The interactive mode can be used to handle application configurations in an interactive manner. This can be used to add, list, export and import applications in the target environment.
```

Comment:
```suggestion
The interactive mode can be used to handle application configurations in an interactive manner. This can be used to add, list, export, and import applications in the target environment.
```